api-schema/src/main/java/org/stellar/anchor/api/sep/AssetInfo.java

```diff
@@ -50,7 +50,7 @@ public static String makeSep11AssetName(String assetCode, String assetIssuer) {
 
   DepositOperation deposit;
   WithdrawOperation withdraw;
-  SendOperation send;
+  @Deprecated SendOperation send;
```

Comment:
Since we are merging this change into 3.0, should we be removing it instead of deprecating it?